err-urlmatcher/urlmatcher.py

```diff
@@ -44,14 +51,27 @@ def url_matcher(self, msg, match):
 
         html = requests.get(url).text
         readable_article = Document(html).summary()
-        readable_article = TAG_RE.sub('', readable_article)
-        readable_article = WHITESPACE_RE.sub(' ', readable_article)
-        readable_article = readable_article.replace('\n', ' ')
-        readable_article = readable_article.replace('&#13;', '')
+        readable_article = self.text_cleanup(readable_article)
 
         if len(readable_article) > max_len:
             readable_article = readable_article[:max_len] + '...'
 
         readable_title = Document(html).short_title()
 
-        return "> " + url + " > " + readable_title + " > " + readable_article
+        page = MetadataParser(html=html)
+        readable_description = page.get_metadata('description')
+
+        if readable_description is None:
+            readable_description = ''
+
+        readable_description = self.text_cleanup(readable_description)
+
+        description = ''
+        if len(readable_description) > len(readable_article):
+            description = readable_description
+        else:
+            description = readable_article
+
+        return "{} > {} > {}".format(url,
```

Comment:
@mrshu before we merge this, can we try running canary with the newlines for couple of days, just to see how it works?